stdimage/models.py

```diff
@@ -31,31 +31,46 @@ class StdImageFieldFile(ImageFieldFile):
 
     def save(self, name, content, save=True):
         super(StdImageFieldFile, self).save(name, content, save)
-        render_variations = self.field.render_variations
-        if callable(render_variations):
-            render_variations = render_variations(
-                file_name=self.name,
-                variations=self.field.variations,
-                storage=self.storage,
-            )
-        if not isinstance(render_variations, bool):
-            msg = (
-                '"render_variations" callable expects a boolean return value,'
-                ' but got %s'
-                ) % type(render_variations)
-            raise TypeError(msg)
-        if render_variations:
-            self.render_variations()
+
+        # Skip rendering if render_variations set False
+        if isinstance(self.field.render_variations, bool) \
```

Comment:
Confusing negation. Why don't you execute render_field_variations only if your statement is true?